core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2207,14 +2207,27 @@ private[spark] object Utils extends Logging {
   }
 
   /**
+   * Return whether it is local mode
+   */
+  def isLocal(conf: SparkConf): Boolean = {
```

Comment:
This needs a better name and description. It says whether the master is local. Use `@return`